SPECS/rdma-core/rdma-core.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        RDMA core userspace libraries and daemons
 Name:           rdma-core
-Version:        49.1
+Version:        59.0
```

Comment:
Double-checking: was the version bump for this package approved by shiproom since it's a big jump?

Reply:
DOCA-OFED requires rdma-core version 58.0 or above. I have highlighted in the PR description and status updates clearly stating about each change. I can double click on this specific version bump in shiproom meeting. if needed.